Sources/Differ/Diff+AppKit.swift

```diff
@@ -34,23 +89,52 @@ extension NSTableView {
     ///   - isEqual:            A function comparing two elements of `T`
     ///   - deletionAnimation:  Animation type for deletions
     ///   - insertionAnimation: Animation type for insertions
-    ///   - indexPathTransform: Closure which transforms zero-based `IndexPath` to desired `IndexPath`
+    ///   - indexPathTransform: Closure which transforms zero-based `IndexPath` to desired `IndexPath`. Only the `.item` is used, not the `.section`.
+    @available(*, deprecated, message: "Use `animateRowChanges(…rowIndexTransform:)`instead. Deprecated because indexPaths are not used in `NSTableView` rows, just Integer row indices.")
```

Comment:
Is is possible to use the "rename" variant of the availability flag here so that Xcode offers a fix-it? For example: 

```swift
@available(*, deprecated, renamed: "loadData")
func fetchData() { }
```

Reply:
Hmm, it's not just a rename, on call site the fixit is probably a great experience because then you'd have your indexPathTransform go into the rowIndexTransform param which won't compile initially. However, this way you wouldn't need to look up the method signature which is a pro.
Given this, should I try it?

Reply:
ping @tonyarnold 🙂

Reply:
No, I think it's fair that this is API breakage, especially given that we're discussing the next release being breaking (version 2.0.0) in #71. I'm happy for this to be merged as-is.